e2e/testdata/fn-eval/simple-function/.expected/config.yaml

```diff
@@ -13,6 +13,6 @@
 # limitations under the License.
 
 testType: eval
-image: gcr.io/kpt-fn/set-namespace:v0.1
+image: set-namespace:v0.1
```

Comment:
could we maybe also add a test where we don't specify the tag?

Reply:
Defaulting tag needs a broader change. We should tag functions with `latest` tag.

@mengqiy

Reply:
`latest` tag will not be supported. See: https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/316/files#diff-5a1e620a21f1c28a342f17f7b625150e976e6d75bc4bc1ba8c24dad36d9249b8R21